quickstarts/install-netobserv.yaml

```diff
@@ -18,14 +18,14 @@ spec:
   tasks:
     - description: |-
         You can install the Network Observability Operator using the OpenShift Container Platform web console Operator Hub. When you install the Operator, it provides the FlowCollector custom resource definition (CRD). You can set specifications in the web console when you create the FlowCollector.
```

Comment:
We should also update `You can install the Network Observability Operator using the OpenShift Container Platform web console Operator Hub` to not mentioning `Operator Hub` , shall we?